components/core/ApplicationUserControls.js

```diff
@@ -103,14 +103,21 @@ const STYLES_ITEM_BOX = css`
 `;
 
 const OpenIntercom = ({ user }) => {
```

Comment:
You should pass in onTogglePopup from the parent component and call it upon click to close the dropdown menu upon selecting "help"

```
<OpenIntercom
  user={this.props.viewer}
  onTogglePopup={this.props.onTogglePopup}
/>

```

```
onClick={() => {
  onTogglePopup();
  show(...);
}}

```